CMakeLists.txt

```diff
@@ -3,6 +3,8 @@
 #on the platform and configuration it is set to build in.
 project(tinyobjloader)
 cmake_minimum_required(VERSION 2.8.6)
```

Comment:
Looking at the CMake documentation, we should probably set the minimum to 2.8.11. I will follow up with a PR.